tests/e2e/testsuites/modify_volume_tester.go

```diff
@@ -57,6 +57,15 @@ func (modifyVolumeTest *ModifyVolumeTest) Run(c clientset.Interface, ns *v1.Name
 	testVolume, _ := volumeDetails.SetupDynamicPersistentVolumeClaim(c, ns, ebsDriver)
 	defer testVolume.Cleanup()
 
+	parameterWithPrefix := func(prefix string, parameters map[string]string) map[string]string {
+		result := make(map[string]string)
+		for key, value := range parameters {
+			result[prefix+key] = value
+		}
+		return result
+	}
+	parametersWithPrefix := parameterWithPrefix("ebs.csi.aws.com/", modifyVolumeTest.ModifyVolumeParameters)
```

Comment:
Would it make more sense to move this logic into `AnnotatePvc`? we can return the result from there and then pass that into `WaitForPvToModify` and so on as needed. We already loop through all the parameters there so I think handling the prefix as well would be easier to reason about and slightly more efficient.

Besides this query, changes lgtm.